src/Http/Http.Abstractions/src/WebSocketManager.cs

```diff
@@ -42,4 +45,22 @@ public virtual Task<WebSocket> AcceptWebSocketAsync()
     /// <param name="acceptContext"></param>
     /// <returns></returns>
     public virtual Task<WebSocket> AcceptWebSocketAsync(WebSocketAcceptContext acceptContext) => throw new NotImplementedException();
+
+    /// <inheritdoc />
+    public override string ToString()
+    {
+        return IsWebSocketRequest switch
+        {
+            false => "IsWebSocket: False",
+            true => $"IsWebSocket: True, RequestedProtocols: {string.Join(", ", WebSocketRequestedProtocols)}",
```

Comment:
```suggestion
            false => "IsWebSocketRequest: False",
            true => $"IsWebSocketRequest: True, RequestedProtocols: {string.Join(",", WebSocketRequestedProtocols)}",
```
IMO `IsWebSocketRequest` makes more sense. Also I removed space from the protocols separator so it's easier to tell new properties vs new items apart.